scripts/developer_tools/lib/commit_and_push.py

```
@@ -69,12 +69,16 @@ def has_staged_changes() -> bool:
 
 def get_staged_diff() -> str:
     """Return the diff of staged changes."""
-    result = subprocess.run(
-        ["git", "diff", "--cached"],
-        capture_output=True,
-        text=True,
-        check=True,
-    )
+    try:
+        result = subprocess.run(
+            ["git", "diff", "--cached"],
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)